### ISSUE TITLE
Feature Request: Add client name for redis_exporter connection

### ISSUE THREAD

**Issue by tinproject** [oliver006/redis_exporter#309]
**Description**
With the inclusion of client list metrics by #293 we can have a list of the clients.

Add client name for redis_exporter connection, so we can filter it out of the list on queries or dashboards.

**What version of redis_exporter are you running?**
Please run `redis_exporter --version` if you're not sure what version you're running.
[ ] 0.3x.x
[X] 1.x.x

**Current behavior**
```
# HELP redis_connected_clients_details Details about connected clients
# TYPE redis_connected_clients_details gauge
redis_connected_clients_details{age="0",cmd="client",db="0",flags="N",host="127.0.0.1",idle="0",name="",port="35308"} 1
```

**Expected behavior**
```
# HELP redis_connected_clients_details Details about connected clients
# TYPE redis_connected_clients_details gauge
redis_connected_clients_details{age="0",cmd="client",db="0",flags="N",host="127.0.0.1",idle="0",name="redis_exporter",port="35308"} 1
```

**Comment by oliver006** [comment]
Makes sense. This should be a easy PR if someone else is interested.
If not then I'll try to add it some time in the next few days.

**Comment by oliver006** [comment]
Addressing this as part of #311

**Comment by oliver006** [comment]
Released as v1.1.1
